PowerArubaSW/Public/RADIUS.ps1

```diff
@@ -0,0 +1,307 @@
+        [PSObject]$connection=$DefaultArubaSWConnection
+    )
+
+    Begin {
+    }
+
+    Process {
+
+        $uri = "rest/v4/radius_servers"
+
+        $conf = New-Object -TypeName PSObject
+
+        $ip = New-Object -TypeName PSObject
+
+        $ip | Add-Member -name "version" -MemberType NoteProperty -Value "IAV_IP_V4"
+
+        $ip | Add-Member -name "octets" -MemberType NoteProperty -Value $address
+
+        $conf | Add-Member -name "address" -membertype NoteProperty -Value $ip
+
+        $conf | Add-Member -name "shared_secret" -MemberType NoteProperty -Value $shared_secret
+
+        if ($PsBoundParameters.ContainsKey('authentication_port')) {
+            $conf | Add-Member -name "authentication_port" -membertype NoteProperty -Value $authentication_port
+        }
+
+        if ($PsBoundParameters.ContainsKey('accounting_port')) {
+            $conf | Add-Member -name "accounting_port" -membertype NoteProperty -Value $accounting_port
+        }
+
+        if ( $PsBoundParameters.ContainsKey('is_dyn_authorization_enabled') ) {
+            if ( $is_dyn_authorization_enabled ) {
+                $conf | Add-Member -name "is_dyn_authorization_enabled" -membertype NoteProperty -Value $true
+            }
+            else {
+                $conf | Add-Member -name "is_dyn_authorization_enabled" -membertype NoteProperty -Value $false
+            }
+        }
+
+        if ($PsBoundParameters.ContainsKey('time_window_type')) {
+            $conf | Add-Member -name "time_window_type" -membertype NoteProperty -Value $time_window_type
+        }
+
+        if ($PsBoundParameters.ContainsKey('time_window')) {
+            $conf | Add-Member -name "time_window" -membertype NoteProperty -Value $time_window
+        }
+
+        if ( $PsBoundParameters.ContainsKey('is_oobm') ) {
+            if ( $is_oobm ) {
+                $conf | Add-Member -name "is_oobm" -membertype NoteProperty -Value $true
+            }
+            else {
+                $conf | Add-Member -name "is_oobm" -membertype NoteProperty -Value $false
+            }
+        }
+
+        $response = Invoke-ArubaSWWebRequest -method "POST" -body $conf -uri $uri -connection $connection
+
+        $run = $response | ConvertFrom-Json
+
+        $run
+    }
+
+    End {
+    }
+}
+
+function Set-ArubaSWRadius {
+
+    <#
+        .SYNOPSIS
+        Set a Radius server
+        .DESCRIPTION
+        Set a Radius server parameters
+        .EXAMPLE
+        Set-ArubaSWRadius -id 1 -address 192.0.2.1 -shared_secret powerarubasw
+        Change parameters for a radius server
+        .EXAMPLE
+        Set-ArubaSWRadius -id 2 -address 192.0.2.2 -shared_secret powerarubasw -authentication_port 1812 -accounting_port 1813 -is_dyn_autorization_enabled -time_window_type TW_PLUS_OR_MINUS_TIME_WINDOW -time_window 0 -is_oobm
+        CHange all the parameters for a radius server, with dynamic autorization and oobm enable
+    #>
+
+    Param(
+        [Parameter (Mandatory = $true)]
```

Comment:
Can you add support of pipeline ? Get-ArubaSWRadius -id X | Set-ArubaSWRadius -address X.X.X.X